src/pages/users/users.tsx

```diff
@@ -135,84 +131,97 @@ export const UsersPage: React.FC = () => {
         const item = row?.original;
         return (
           <div className="font-medium text-sm">
-            <RenderIf condition={item?.status?.toLowerCase() === "pending"}>
-              <p className="text-amber">{item?.status}</p>
+            <RenderIf condition={item?.status === 0}>
+              <p className="text-amber">Pending</p>
             </RenderIf>
-            <RenderIf condition={item?.status?.toLowerCase() === "approved"}>
-              <p className="text-green-base">{item?.status}</p>
+            <RenderIf condition={item?.status === 1}>
+              <p className="text-green-base">Active</p>
             </RenderIf>
-            <RenderIf condition={item?.status?.toLowerCase() === "suspended"}>
-              <p className="text-accent-primary">{item?.status}</p>
+            <RenderIf condition={item?.status === 2}>
+              <p className="text-accent-primary">Suspended</p>
             </RenderIf>
           </div>
         );
       },
     },
   ];
 
-  const [page, setPage] = useState(1);
-  const [itemsPerPage] = useState(10);
-
-  const [searchParams, setSearchParams] = useState("");
-
-  const handlePageChange = (page: number) => {
-    setPage(page);
-    setPaginationParams(page, itemsPerPage, searchParams, setSearchParams);
-  };
+  useEffect(() => {
+    getPaginationParams(location, setPage, () => {});
+  }, [location, setPage]);
 
   const navigate = useNavigate();
 
   return (
     <div className="flex flex-col gap-5 px-4 pt-3 md:pt-5 pb-5 md:pb-10 view-page-container overflow-y-scroll">
-      <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-4 gap-4 pb-6">
-        {userStatistics.map((item) => (
-          <div
-            key={item.id}
-            className="flex items-center gap-4 p-4 rounded-lg bg-light-blue-4"
-          >
-            <div className="grid place-content-center rounded-full size-12 bg-light-blue-3">
-              <Icon
-                icon="lucide:users"
-                className="size-6 text-text-secondary"
-              />
-            </div>
-            <div className="grid gap-1">
-              <h1 className="text-sm text-text-secondary">{item.label}</h1>
-              <p className="text-2xl text-text-primary">{item.value}</p>
+      <RenderIf condition={isLoading || isLoadingCount || isLoadingStatistics}>
+        <div className="flex w-full h-96 items-center justify-center">
+          <Loader className="spinner size-6 text-green-1" />
+        </div>
+      </RenderIf>
```

Comment:
@AdejokeOgunyinka We want our loader to show only when the table needs to be updated. Move the loader to where the table is.